molgenis-security-ui/src/main/resources/l10n/security-ui_en.properties

```diff
@@ -14,7 +14,9 @@ security-ui-member-remove=Remove
 security-ui-group-attribute-name-placeholder=my-group
 security-ui-add-group=Add Group
 security-ui-members-header=Members
-security-ui-group-attribute-name-taken-message=This group name is not available any more, please choose a different name.
```

Comment:
Renaming the key will break the current UI, is this intended?

Side note: In my opinion 'name' is a better description than 'identifier' since that is how the metadata defines these terms. Regardless of how the UI calls it in English.